jobserver/templates/error.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'base-tw.html' %}
+
+{% block content %}
+<div class="min-h-full px-4 py-16 sm:px-6 sm:py-24 md:grid md:place-items-center lg:px-8">
+  <div class="sm:flex">
+    <p class="text-4xl font-extrabold text-oxford-600 sm:text-5xl">{{ error_code }}</p>
+    <div class="sm:ml-6">
+      <div class="sm:border-l sm:border-slate-200 sm:pl-6">
+        <h1 class="text-4xl font-extrabold text-slate-900 tracking-tight sm:text-5xl">{{ error_name }}</h1>
+        <p class="mt-2 text-base text-slate-500">{{ error_message }}</p>
```

Comment:
Technically, this text isn't quite lining up with the title and the buttons. I don't know if it's worth fixing though

Reply:
Re-reading the code today, it was overly complex for what I was trying to achieve. Pushing an update shortly which should align everything a bit better.

Reply:
Pushed 🚀